Emby.Server.Implementations/Session/SessionManager.cs

```diff
@@ -147,28 +173,24 @@ protected virtual void Dispose(bool disposing)
 
             if (disposing)
             {
-                // TODO: dispose stuff
+                _idleTimer?.Dispose();
             }
 
+            _idleTimer = null;
```

Comment:
shouldn't this be done in `if (disposing)`?

Reply:
No